Gemfile

```diff
@@ -20,7 +20,6 @@ gem 'sentry-ruby'
 gem 'sqlite3'
 
 group :development do
-  gem 'debase'
   gem 'ruby-debug-ide'
```

Comment:
Do we still need gem 'ruby-debug-ide' in this case? Maybe remove all development gems?

Reply:
I think they can be removed. Pushed commit ac08980a2eff01884a4fd396cab157f79e22fb94 to do so.